policies/templates/gcp_sql_allowed_authorized_networks_v1.yaml

```diff
@@ -71,9 +71,14 @@ spec:
             	asset.asset_type == "sqladmin.googleapis.com/Instance"
             
             	allowed_authorized_networks = lib.get_default(params, "authorized_networks", [])
-            	configured_networks := {network |
-            		network = asset.resource.settings.ipConfiguration.authorizedNetworks[_].value
-            	}
+            	
+                # Check whether authorizedNetworks field exists, so that
+                # we can report violation when this field is not set
+                config_auth_networks = lib.get_default(asset.resource.data.settings.ipConfiguration, "authorizedNetworks", [{"value":"authorized network unspecified"}])
```

Comment:
Formatting looks off. Please run `make format`.

Reply:
File is now formatted.